charts/nexus-repository-manager/README.md

```diff
@@ -118,16 +114,16 @@ The following table lists the configurable parameters of the Nexus chart and the
 | Parameter                                  | Description                                                                                  | Default                                                                                                                                         |
 |--------------------------------------------|----------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
 | `deploymentStrategy`                       | Deployment Strategy                                                                          | `Recreate`                                                                                                                                      |
-| `nexus.imagePullPolicy`                    | Nexus Repository image pull policy                                                                      | `IfNotPresent`                                                                                                                                  |
-| `nexus.imagePullSecrets`                   | Secret to download Nexus Repository image from private registry                                         | `nil`                                                                                                                                           |
+| `nexus.imagePullPolicy`                    | Nexus Repository image pull policy                                                           | `IfNotPresent`                                                                                                                                  |
+| `imagePullSecrets`                         | The names of the kubernetes secrets with credentials to login to a registry                  | `[]`                                                                                                                                            |
```

Comment:
this is the only addition to the table that's new. the rest of the changes are mere whitespace.